### PR TITLE
Integration test for LPS-188504

### PR DESCRIPTION
**Pull request by kildarep**
Related: https://github.com/brianchandotcom/liferay-portal/pull/139497

### CODE REVIEW

File: modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectFieldLocalServiceTest.java [hunk 1]
```diff
@@ -170,6 +170,66 @@ public void testAddCustomObjectField() throws Exception {
 			() -> _addCustomObjectDefinitionWithEncryptedObjectField(
 				"AES", true, "",
 				ObjectDefinitionConstants.STORAGE_TYPE_DEFAULT));
+		AssertUtils.assertFailure(
+			ObjectFieldBusinessTypeException.class,
+			"Salesforce storage type does not aggregation and attachment " +
```
Comment (carolmariaabb) [inline modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectFieldLocalServiceTest.java:175]: @kildarep the message seems incomplete. Wouldn't it be "Salesforce storage type does not **support** aggregation and attachment..."?

File: modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectFieldLocalServiceTest.java [hunk 2]
```diff
@@ -170,6 +170,66 @@ public void testAddCustomObjectField() throws Exception {
...
+							new ObjectFieldSettingBuilder(
+							).name(
+								"objectRelationshipName"
+							).value(
+								"oneToManyRelationshipName"
+							).build())
+					).build())));
+		AssertUtils.assertFailure(
+			ObjectFieldBusinessTypeException.class,
+			"Salesforce storage type does not aggregation and attachment " +
```
Comment (carolmariaabb) [inline modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectFieldLocalServiceTest.java:214]: @kildarep same as above

**Review by carolmariaabb — CHANGES_REQUESTED** [review]
@kildarep, greate job 🎉. 

Just a note regarding the error message.

Thanks!

File: modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectFieldLocalServiceTest.java (hunk 1 above)
Comment (carolmariaabb) [reply modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectFieldLocalServiceTest.java:175]: @kildarep could you please update the message? Thanks!

Comment (kildarep) [reply modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectFieldLocalServiceTest.java]: Updated messages. Thanks!

**Comment by carolmariaabb** [comment]
ci:test:relevant

**Comment by carolmariaabb** [comment]
ci:forward:force

**Pull request closed without merging**